cfpb/cfgov-refresh
# Side nav

## Additions

- Provided option to exclude sibling pages in secondary navigation for Browse/Filterable Pages

## Testing

- Import Refresh data
- Navigate to `content.localhost:8000/cfpb-ombudsman/` verify secondary nav by default shows all sibling links
![screen shot 2016-03-10 at 11 39 14 am](https://cloud.githubusercontent.com/assets/254877/13676833/651d3e66-e6b5-11e5-9646-cf67ff468668.png)
- Edit Sidebar `http://localhost:8000/admin/pages/468/edit/` and check to box to exclude sibling links
![screen shot 2016-03-10 at 11 39 31 am](https://cloud.githubusercontent.com/assets/254877/13676850/76a3ac06-e6b5-11e5-9c0a-da8062cdbc8c.png)
- Share page and verify siblings are no longer rendered on  `content.localhost:8000/cfpb-ombudsman/`
![screen shot 2016-03-10 at 11 38 57 am](https://cloud.githubusercontent.com/assets/254877/13676869/8820074a-e6b5-11e5-9735-bcf94df58b65.png)



## Review

- @kurtw 
- @richaagarwal 


## Checklist
